torch/onnx/symbolic_opset9.py

```diff
@@ -2087,3 +2090,16 @@ def _weight_norm(g, weight_v, weight_g, dim):
         return g.op("Mul", div, weight_g)
     else:
         return g.op("ATen", weight_v, weight_g, dim_i=dim, operator_s="_weight_norm")
+
+# Ops below are for PyTorch Quantization conversion process.
+@parse_args('v', 'f', 'i', 't')
```

Comment:
ditto, let's define this symbolic in symbolic_caffe2.py, and register it there.

Reply:
Same reason as relu for why I couldn't do this. These ops below are in the `aten` namespace